### PR TITLE
Handheld explosives nerf, or; The saltiest of the salt nerfs yet.

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
China lake, Lenar and the other pump action GL now have slowdown, they also now have fire delay. No more spam-blasting the entire mag in a few seconds.

Emperor is gone, it is dead. no more gyrojet. it's GONE.

payload shells are DEAD they are GONE.

artist revolvers may no longer roll gyrojet or GL as valid ammo types.

 rapidfire explosions have been a cancer used by powergamers for basically the entire lifetime of the server and i should have done this sooner.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: maps/_DeepForest/map/_Deep_Forest.dmm [hunk 1]
```diff
@@ -15418,7 +15418,7 @@
 "keY" = (
 /obj/structure/closet/crate/secure/plasma,
 /obj/item/ammo_magazine/ammobox/shotgun/stunshells,
-/obj/item/ammo_casing/shotgun/payload/prespawned,
+//obj/item/ammo_casing/shotgun/payload/prespawned,
```
Comment (Trilbyspaceclone) [inline maps/_DeepForest/map/_Deep_Forest.dmm:15421]: dont do this for mapping, code in map is bad

File: maps/SpaceFortress/map/exc_fort.dmm [hunk 2]
```diff
@@ -54,7 +54,7 @@
 /obj/item/ammo_magazine/ammobox/shotgun/buckshot,
 /obj/item/ammo_magazine/ammobox/shotgun/incendiary,
 /obj/item/ammo_magazine/ammobox/shotgun/incendiary,
-/obj/item/ammo_magazine/ammobox/shotgun/payload,
+//obj/item/ammo_magazine/ammobox/shotgun/payload,
```
Comment (Trilbyspaceclone) [inline maps/SpaceFortress/map/exc_fort.dmm:57]: code in map is infact bad

File: code/modules/mob/living/carbon/superior_animal/human/subtype/voidwolf/voidwolf.dm [hunk 3]
```diff
@@ -301,15 +301,15 @@
 	icon_state = "reaver_gyro"
 	projectilesound = 'sound/weapons/guns/fire/ubgl.ogg'
 	projectiletype = /obj/item/projectile/bullet/gyro
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/carbon/superior_animal/human/subtype/voidwolf/voidwolf.dm:303]: Why are we keeping a mob that uses it? maybe swap this to an RPG?

**Comment by cdb-is-not-good** [comment]
map spawns hard removed instead of commented, and the gyro reaver is commented out fully now. Later on will add idk an RPG reaver or a china lake reaver.

File: maps/_DeepForest/map/_Deep_Forest.dmm [hunk 4]
```diff
@@ -22121,9 +22121,9 @@
 /area/colony/exposedsun/pastgate)
 "oCD" = (
 /obj/structure/safe,
-/obj/item/gun/projectile/gyropistol,
-/obj/item/ammo_magazine/a75,
-/obj/item/ammo_magazine/a75,
+//obj/item/gun/projectile/gyropistol,
```
Comment (Trilbyspaceclone) [inline maps/_DeepForest/map/_Deep_Forest.dmm:22124]: Still not removed

Comment (cdb-is-not-good) [reply maps/_DeepForest/map/_Deep_Forest.dmm:22124]: fix'd

File: maps/_DeepForest/map/_Deep_Forest.dmm [hunk 5]
```diff
@@ -15418,7 +15418,7 @@
 "keY" = (
 /obj/structure/closet/crate/secure/plasma,
 /obj/item/ammo_magazine/ammobox/shotgun/stunshells,
-/obj/item/ammo_casing/shotgun/payload/prespawned,
```
Comment (Trilbyspaceclone) [inline maps/_DeepForest/map/_Deep_Forest.dmm:15421]: This still isnt fixed its commited out rather then proper removed

Comment (cdb-is-not-good) [reply maps/_DeepForest/map/_Deep_Forest.dmm:15421]: fixed

**Comment by benj8560** [comment]
can I suggest simply leaving the funny gyrojet gun in as a adminbus/ebent gun or something.

**Comment by benj8560** [comment]
otherwise good PR. hate explosive meta. Fuck it. begon.

**Comment by cdb-is-not-good** [comment]
> can I suggest simply leaving the funny gyrojet gun in as a adminbus/ebent gun or something.

it's only commented out, we can maybe look at undoing that for a specific use if that comes up.

**Pull request merged**